static/js/apps/nl_interface/query_result.tsx

```diff
@@ -128,7 +128,18 @@ export const QueryResult = memo(function QueryResult(
                 : null,
           });
         } else {
-          setErrorMsg("Sorry, we couldn't answer your question.");
+          // If there was no place recognized, we might end up with 0
+          // categories.  Check if that's the case, and provide a different
+          // error message.
+          if ("placeSource" in resp.data && resp.data["placeSource"]) {
+            setErrorMsg(
+              "Could not recognize any place in the query! " +
+                "If you included a place, it is possible " +
+                "we don't recognize it yet!"
```

Comment:
nit: this sounds like we have a a set of those places we recognize vs those that we don't etc. That could be a bit problematic to communicate to users. Could we simply state that "we failed to recognize it" (which is also an accurate reflection)?

Reply:
Ok, done.  I wasn't sure if we wanted to suggest adding a place, but we can revisit.